merlin-server/sql/merlin/merge_request.sql

```diff
@@ -0,0 +1,85 @@
+create type merge_request_status as enum ('pending', 'in-progress','accepted', 'rejected', 'withdrawn');
+
+create table merge_request(
+      id integer generated always as identity
+        primary key,
+      plan_id_receiving_changes integer,
+      snapshot_id_supplying_changes integer,
+      merge_base_snapshot_id integer not null,
+      status merge_request_status default 'pending',
+      requester_username text not null,
+      reviewer_username text not null default ''
```

Comment:
I'm curious about pros/cons of using the empty string as the default here, rather than `null`.

Reply:
This was mostly a style choice instead of a pros/cons choice, as we can get equivalent behavior with either `reviewer_username text not null default ''` or `reviewer_username text default null`. The main two differences I can think of are:
- Whoever works with the data is guaranteed that getting `NULL` means that something's wrong
- `reviewer_username` can never go from having a value to `NULL`.

Reply:
Gotcha - thanks for clarifying :+1: I would suggest we use `null` rather than an empty string to mean "no data" here - we used a similar pattern for the `reason` field in simulation_dataset (which is always null unless the simulation `status` is `failed`):
https://github.com/NASA-AMMOS/aerie/blob/c834e9c05d21b32cd07f5018c2ccb47ad39de94a/merlin-server/sql/merlin/tables/simulation_dataset.sql#L21

Reply:
Ah, if we're already using that pattern, then sure, I can change it to `reviewer_username text default null`.

Reply:
Done